Log log
---
int32 status
string message